build.gradle.kts

```diff
@@ -112,6 +112,20 @@ dependencies {
     implementation("org.apache.kafka:kafka_2.13:${Versions.kafka}", excludeLog4j)
     implementation("io.confluent:kafka-avro-serializer:${Versions.confluent}", excludeLog4j)
     implementation("io.confluent:kafka-schema-registry:${Versions.confluent}", excludeLog4j)
+    constraints {
+        implementation("org.yaml:snakeyaml") {
+            because("io.confluent:kafka-schema-registry:${Versions.confluent} -> https://advisory.checkmarx.net/advisory/vulnerability/CVE-2022-25857/")
+            version {
+                require("1.31")
+            }
+        }
+        implementation("org.glassfish:jakarta.el") {
+            because("io.confluent:kafka-schema-registry:${Versions.confluent} -> https://advisory.checkmarx.net/advisory/vulnerability/CVE-2021-28170/")
+            version {
+                require("3.0.4")
+            }
+        }
+    }
```

Comment:
Er dette for å enforce versjoner på transitive/underliggende dependencies?

Reply:
Ja, det er for å overstyre transitive avhengigheter (er nok mer enn en måte å gjøre det på, men denne er ganske ryddig vet at får dokumentert noen nyttige detaljer).